firstmatch/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Google LLC
```

Comment:
Could we use a more meaningful name for this sub project? Either `demo` or `getstarted` makes sense to me since this was installed using the previous `02-open-match-demo.yaml` under the `Getting Started` section in our website. 

I guess first match is fine as long as we keep the naming in sync with our website

Reply:
I think Getting Started will have multiple pages instead of being just first match soon?

Reply:
Until we have more things to put under a getting started folder, I prefer to keep this as a top level item.